src/components/layouts/FilterBar.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import { Box, Stack, Typography } from '@mui/material'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { useEffect, useState } from 'react'
+import SearchBar from '../searchBar/SearchBar'
```

Comment:
A better file layout pattern could also be to rename the files to `...components/Searchbar/index.tsx`. This way you can import it using a shorter path like:

```ts
import SearchBar from '@/app/components/Searchbar`
```
instead of
```ts
import SearchBar from '@/app/components/searchBar/SearchBar'
```

Reply:
It's also good convention in typescript to prefer named exports instead of default exports

See: https://dev.to/phuocng/avoid-using-default-exports-a1c